numpy/core/fromnumeric.py

```diff
@@ -1141,9 +1142,10 @@ def argmin(a, axis=None, out=None):
     Examples
     --------
     >>> a = np.arange(6).reshape(2,3)
+    >>> a += 10
```

Comment:
A nit: I'd combine this with the previous line. The user probably isn't interested in how we construct the array used for the example, so no need to split the line into pieces

Reply:
Yeah I was unsure myself. Went with this approach in the end because I thought 3 operations in a line are a bit too much. Should I change it?